.github/workflows/testing_apps.yml

```diff
@@ -0,0 +1,58 @@
+name: Testing apps
+on: [push, pull_request]
+
+jobs:
+  testing_app:
+    strategy:
+      fail-fast: false
+      matrix:
+        test_app:
+          - name: npm
+            working_directory: test_apps/npm
+            script: |
+              npm install --ci
+              npm add --save-dev ../../webpack-encore.tgz
```

Comment:
Installing the `webpack-encore.tgz` (packed in a previous step before) prevent the package manager to fails if checksums are different, which was the case for npm/pnpm/yarn pnp.